boxsdk/object/folder.py

```diff
@@ -382,3 +383,45 @@ def delete(self, recursive=True, etag=None):
         :raises: :class:`BoxAPIException` if the specified etag doesn't match the latest version of the folder.
         """
         return super(Folder, self).delete({'recursive': recursive}, etag)
+
+
+    def create_web_link(self, target_url, name=None, description=None):
+        """
+        Create a WebLink with a given url.
+
+        :param url:
+            The url the web link points to.
+        :type url:
+            `unicode`
+        :param parent_id:
+            The id of the parent folder where the web link is created.
+        :type parent_id:
+            `unicode`
+        :param name:
+            The name of the web link. Defaults to the URL is not specified.
+        :type name:
+            `unicode`
+        :param description:
+            Description of the web link
+        :type name:
+            `unicode`
+        :return:
+            A :class:`WebLink` object with the given web link id.
+        :rtype:
+            :class:`WebLink`
+        """
+        url = '{0}/web_links'.format(API.BASE_API_URL)
+        web_link_attributes = {
+            'url': target_url,
+            'parent': {
+                'id': self.object_id
+            }
+        }
+        if name is not None:
+            web_link_attributes['name'] = name
+        if description is not None:
+            web_link_attributes['description'] = description
+        box_response = self._session.post(url, data=json.dumps(web_link_attributes))
```

Comment:
Use `self.update_info()` instead.

Reply:
@jmoldow, when I switch to self.update_info() it give me a pylint error saying BaseObject has no json member

Reply:
That's because you need to change more than just this one line. You would get rid of this line and the next two lines, and just have

```python
return self.update_info(web_link_attributes)
```